src/component/mxgraph/config/StyleConfigurator.ts

```diff
@@ -305,7 +307,7 @@ export default class StyleConfigurator {
         if (markers.length > 0) {
           styleValues.set(StyleIdentifier.BPMN_STYLE_MARKERS, markers.join(','));
         }
-      } else if (bpmnElement.kind === ShapeBpmnElementKind.POOL) {
+      } else if (ShapeUtil.isPoolOrLane((bpmnElement as ShapeBpmnElement).kind)) {
         // mxConstants.STYLE_HORIZONTAL is for the label
         // In BPMN, isHorizontal is for the Pool
```

Comment:
:question: I suggest to update the comment
```suggestion
        // In BPMN, isHorizontal is for the Shape
```